src/utils/helpers/componentHelpers.ts

```diff
@@ -0,0 +1,21 @@
+import type { KeyboardEvent, RefObject } from 'react';
+import type { ChannelStatus } from '../types/common';
+import type { SubmitFormType } from '../types/form';
+
+export const submitFormOnKeyboardPress = (
+  event: KeyboardEvent,
+  submitForm: SubmitFormType,
+  channelStatus?: ChannelStatus
+): void => {
+  if (!!channelStatus && channelStatus !== 'received') return;
+  if (event.ctrlKey && event.code === 'Enter') {
```

Comment:
I would propose to create constants for the key codes =)

Reply:
Done!